roles/nginxplus/templates/logrotate.d.app_protect.j2

```diff
@@ -0,0 +1,12 @@
+# logrotate configuration for app_protect in {{ ansible_managed | comment }}
```

Comment:
Could we generalize this into a true template? with a variable for the path/file to be rotated? Then we might be able to handle log rotation using group vars . . .